invenio_rdm_records/resources/serializers/bibtex/schema_formats.py

```diff
@@ -81,7 +81,7 @@ def publication_article():
         name = "article"
         req_fields = ["author", "title", "journal", "year"]
         opt_fields = ["volume", "number", "pages", "month", "note"]
-        return format(name, req_fields, opt_fields)
+        return BibTexFormatter.format(name, req_fields, opt_fields)
```

Comment:
Fixed a small issue where `format` was the in-built format method and not `BibTexFormatter.format`